worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -310,6 +321,12 @@ class PushDataHandler extends BaseMessageHandler with Logging {
       workerSource.startTimer(WorkerSource.SlavePushDataTime, key)
     }
 
+    // For test
+    if (conf.testPushDataTimeout && !PushDataHandler.pushDataTimeoutTested) {
```

Comment:
why handlePushMergedData use variable in pushDataHandler instead of the variable in object  PushDataHandler as handlePushData, it looks a bit confused here.

Reply:
because I want the variable process-level instead of thread local